packages/build-utils/test/unit.get-project-paths.test.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import { normalizePath } from '../src';
+import { getProjectPaths, ProjectPath } from '../src/get-project-paths';
+import { FixtureFilesystem } from './utils/fixture-filesystem';
+
+describe.each<[ProjectPath[], number, number, string, ProjectPath[]?]>([
+  [[], 2, 4, '32-monorepo-highly-nested'],
+  [['backend/app-three'], 2, 3, '33-hybrid-monorepo', ['frontend']],
+  [
+    ['backend/app-three', 'frontend/app-one', 'frontend/app-two'],
+    3,
+    6,
+    '34-monorepo-no-workspaces',
+  ],
+  [[], 1, 1, '35-no-monorepo'],
+  [['frontend/app-two'], 2, 4, '36-monorepo-some-nested'],
+])(
```

Comment:
Can we swap the order so the test fixture name is first?

Similar to this file:

https://github.com/vercel/vercel/blob/main/packages/build-utils/test/unit.detect-workspace-managers.test.ts#L6